backend/src/main/java/ru/hh/career/solution/resource/AdviserResource.java

```diff
@@ -33,31 +34,43 @@ public AdviserResource(AdviserService adviserService) {
   }
 
   @GET
-  public PageResponseDto getAdvisers(@DefaultValue("20") @QueryParam("per_page") Integer perPage,
+  public PageResponseDto getAdvisers(@DefaultValue("100") @QueryParam("per_page") Integer perPage,
                                      @DefaultValue("0") @QueryParam("page") Integer page) {
-
-    return new PageResponseDto(adviserService.getAdvisers(perPage, page).stream().
-      map(AdviserMapper::map).
+    Long countAdvisers = adviserService.getCountAdvisers();
+    List<Adviser> adviserList = adviserService.getAdvisers(perPage, page);
+    return new PageResponseDto(adviserList.stream().
+      map(AdviserMapper::mapToAdviserDto).
       collect(Collectors.toList()),
```

Comment:
А почему бы сразу не сделать лист дто? Просто adviserList тебе по сути не нужен, только как промежуточная коллекция под стрим. И стрим в качестве аргумента выглядит не оч, легче читать, когда аргументы просто объекты
```suggestion
    List<AdviserDto> advisers = adviserService.getAdvisers(perPage, page).stream().
      map(AdviserMapper::mapToAdviserDto).
      collect(Collectors.toList());
    return new PageResponseDto(advisers,
```